processor/elasticapmprocessor/testdata/ecs/elastic_metric/output.yaml

```diff
@@ -0,0 +1,327 @@
+resourceMetrics:
+  - resource:
+      attributes:
+        - key: agent.activation_method
+          value:
+            stringValue: automatic
+        - key: agent.ephemeral_id
+          value:
+            stringValue: ephem-123
+        - key: agent.name
+          value:
+            stringValue: ElasticAPM
+        - key: agent.version
+          value:
+            stringValue: 1.0.0
+        - key: client.address
+          value:
+            stringValue: 192.168.1.100
+        - key: client.port
+          value:
+            intValue: "8080"
+        - key: cloud.account.id
+          value:
+            stringValue: "123456789012"
+        - key: cloud.account.name
+          value:
+            stringValue: test-account
+        - key: cloud.availability_zone
+          value:
+            stringValue: us-east-1a
+        - key: cloud.instance.id
+          value:
+            stringValue: i-1234567890abcdef0
+        - key: cloud.instance.name
+          value:
+            stringValue: test-instance
+        - key: cloud.machine.type
+          value:
+            stringValue: t3.medium
+        - key: cloud.origin.account.id
+          value:
+            stringValue: "987654321098"
+        - key: cloud.origin.provider
+          value:
+            stringValue: azure
+        - key: cloud.origin.region
+          value:
+            stringValue: westus
+        - key: cloud.origin.service.name
+          value:
+            stringValue: compute
+        - key: cloud.platform
+          value:
+            stringValue: aws_ec2
+        - key: cloud.project.id
+          value:
+            stringValue: project-123
+        - key: cloud.project.name
+          value:
+            stringValue: my-project
+        - key: cloud.provider
+          value:
+            stringValue: aws
+        - key: cloud.region
+          value:
+            stringValue: us-east-1
+        - key: container.id
+          value:
+            stringValue: abc123def456
+        - key: container.image.name
+          value:
+            stringValue: myapp
+        - key: container.image.tag
+          value:
+            stringValue: latest
+        - key: container.image.tags
+          value:
+            stringValue: v1.0.0,latest
+        - key: container.name
+          value:
+            stringValue: my-container
+        - key: container.runtime
+          value:
+            stringValue: docker
+        - key: data_stream.dataset
+          value:
+            stringValue: apm.app.test_service
```

Comment:
[for reviewer] note: the service specific data_stream.dataset. This is expected and consolidates the test cases under the delete `elastic_apm`

Reply:
Great switch here, cleaner testing 👍🏼